apiserver/alembic/versions/201711081734_49be2190c22d_add_challenges.py

```diff
@@ -0,0 +1,100 @@
+"""Add challenges
+
+Revision ID: 49be2190c22d
+Revises: 33de9025cc63
+Create Date: 2017-11-08 17:34:20.134831+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+
+# revision identifiers, used by Alembic.
+revision = '49be2190c22d'
+down_revision = '33de9025cc63'
```

Comment:
What is in the repository for alembic and sql schema doesn't currently match production reality, I'll create a pull request shortly to get the repo back in line with reality again. At that point though this down_revision will probably need to be changed to whatever is actually the head revision at that point.